### DISCUSSION TITLE
--exclude flag deprecated?

### DISCUSSION THREAD

**Discussion by umamibeef** [rizsotto/Bear#541] [Q&A]
I'm trying to exclude folders out of the bear command recording, but the flag appears to have been removed in the version I am using. Is there a another (likely newer and better) way for me to exclude folders?

**Comment by rizsotto** [comment]
Hey @umamibeef , not sure which version you are talking about. The latest version still has the functionality to exclude folders or compiler calls, but it's via configuration file. Please read the [man page](https://github.com/rizsotto/Bear/blob/master/source/citnames/man/bear-citnames.1.md) how to do set up one.

**Comment by umamibeef** [comment]
Thanks @rizsotto! That's exactly what I was missing. Have a great day!
